pkg/client/client.go

```diff
@@ -751,13 +751,14 @@ func (c *SPDKClient) EngineBackupStatus(backupName, engineName, replicaAddress s
 	})
 }
 
-func (c *SPDKClient) ReplicaBackupStatus(backupName string) (*spdkrpc.BackupStatusResponse, error) {
+func (c *SPDKClient) ReplicaBackupStatus(backupName, replicaAddress string) (*spdkrpc.BackupStatusResponse, error) {
 	client := c.getSPDKServiceClient()
 	ctx, cancel := context.WithTimeout(context.Background(), GRPCServiceTimeout)
 	defer cancel()
 
 	return client.ReplicaBackupStatus(ctx, &spdkrpc.BackupStatusRequest{
-		Backup: backupName,
+		Backup:         backupName,
+		ReplicaAddress: replicaAddress,
```

Comment:
Suggest using the replica address from the client directly instead of relying on another redundant parameter of replica address.

```go
	return client.ReplicaBackupStatus(ctx, &spdkrpc.BackupStatusRequest{
		Backup: backupName,
		ReplicaAddress: c.serviceURL,
	})
```

Reply:
Updated. Thank you.